elastic/kibana
# Replacing call to 'cat' API with 'indices' API

## Summary

Replacing the calls to `cat` API with the `indices` API and adding a functional test to verify the toast message is shown

Resolves Issues: 
- [116971](https://github.com/elastic/kibana/issues/116971)
- [114049](https://github.com/elastic/kibana/issues/114049)



## Functional Testing Instructions

-  From the top level project folder, run the function test server: `node scripts/functional_tests_server.js --config /Your/project/path/kibana-1/x-pack/test/security_functional/unsecure_cluster_alert.config.ts`

- Then, in another terminal, from the top level project folder, run the test file: ` node scripts/functional_test_runner --config x-pack/test/security_functional/unsecure_cluster_alert.config.ts
`

## Manual Testing Instructions

- Start ES locally with security disabled: `yarn es snapshot -E xpack.security.enabled=false`
- Start Kibana locally with: `yarn start`
- Navigate to Dev Tools > Consol, create an index, and add at least 1 doc to that index
 - Create index: `PUT /my-index-000001`
 - Add a doc: 
 `POST my-index-000001/_doc/
{
  "@timestamp": "2099-11-15T13:12:00",
  "message": "GET /search HTTP/1.1 200 1070000",
  "user": {
    "id": "user1"
  }
}`
- Refresh the browser
- A toast message should appear on the bottom right of the UI alerting you that the cluster is not using security
